tests/base/bounding_box_3.output

```diff
@@ -65,7 +65,7 @@ DEAL::Is neighbor G with A: 3
 DEAL::Vice-versa: 3
 DEAL::Is neighbor G with B: 3
 DEAL::Vice-versa: 3
-DEAL::Is neighbor G with C: 0
+DEAL::Is neighbor G with C: 3
```

Comment:
Why does the test output change?

Reply:
The boxes C [0,2] and G [0.8,1.8] are mergable neighbors (3). Currently, it is considered no neighbor (0).

Reply:
Ah, thanks!